src/elements/global-header/global-header.js

```diff
@@ -1,18 +1,13 @@
 import Avatar from '@app-elements/avatar'
-import connect from '@app-elements/connect'
 import { Link } from '@app-elements/router'
+import { useMappedState } from '@app-elements/use-mapped-state'
+
+import store from '/store'
```

Comment:
If we use a function and not a const for the component definition, we can access `this.context`, which I find nice because I can avoid importing `store` on occasion 😅